requirements/conda.txt

```diff
@@ -1,4 +1,4 @@
-spyder>=4.0.0
+spyder>=4.1.0
```

Comment:
@ccordoba12 should we add an upper bound?

`spyder>=4.1.0<5`

Reply:
Yes, I agree. @steff456, please address @goanpeca's suggestion.